### PR TITLE
Introduce Play interface

### PR DESCRIPTION
**Pull request by anelook**
There is a built-in interface which comes with ClickHouse setup -"/play". Added information how to access it it into an article

### CODE REVIEW

File: docs/products/clickhouse/howto/use-play.rst [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Use ClickHouse® play UI
```
Comment (dewan-ahmed) [inline docs/products/clickhouse/howto/use-play.rst:1]: A suggestion to use Aiven for ClickHouse® throughout the document.

File: docs/products/clickhouse/howto/use-play.rst [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+========================
+
+ClickHouse® includes a built-in user interface for running SQL queries. You can access it through HTTPS protocol. Follow these steps to use it:
```
Comment (dewan-ahmed) [inline docs/products/clickhouse/howto/use-play.rst:4]: ~You can access it through HTTPS protocol.~ You can access it from a web browser over the HTTPS protocol.

File: docs/products/clickhouse/howto/use-play.rst [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+1. Log in to the `Aiven web console <https://console.aiven.io/>`_ and select your ClickHouse service.
```
Comment (dewan-ahmed) [inline docs/products/clickhouse/howto/use-play.rst:6]: Log in to the `Aiven web console <https://console.aiven.io/>`_, choose the right project, and select your Aiven for ClickHouse® service.

Comment (dewan-ahmed) [reply docs/products/clickhouse/howto/use-play.rst:6]: Note to the author: I'm not sure if we need to mention the full form Aiven for ClickHouse® every time or mentioning it once at the beginning will suffice. 

Also, "choose the right project" part is a soft suggestion so feel free to disregard it.

File: docs/products/clickhouse/howto/use-play.rst [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+#. In the *Overview* tab find *Connection information* and select **ClickHouse HTTPS**
```
Comment (dewan-ahmed) [inline docs/products/clickhouse/howto/use-play.rst:7]: In the *Overview* tab, find *Connection information* and select **ClickHouse HTTPS**.

(a ',' and '.' added).

File: docs/products/clickhouse/howto/use-play.rst [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+#. Copy Service URI and use it to open a page in the browser with the URL ``your-full-service-uri/play``.
```
Comment (dewan-ahmed) [inline docs/products/clickhouse/howto/use-play.rst:8]: Copy the Service URI and navigate to ``SERVICE_URI/play`` from a web broweser.

File: docs/products/clickhouse/howto/use-play.rst [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+#. Set the *name* and the *password* of the user of whose behalf you want to run the queries.
```
Comment (dewan-ahmed) [inline docs/products/clickhouse/howto/use-play.rst:9]: Set the *name* and the *password* of the user ~of~ on whose behalf you want to run the queries.

File: docs/products/clickhouse/howto/use-play.rst [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+#. Enter the body of the query.
+#. Click **Run**.
+
+.. note::
+    `/play` is an alternative to the :doc:`query editor <use-query-editor>`. The query editor can be accessed directly from console to run requests on behalf of the default user.
```
Comment (dewan-ahmed) [inline docs/products/clickhouse/howto/use-play.rst:14]: The query editor can be accessed directly from the console to run requests on behalf of the default user.

File: docs/products/clickhouse/howto/use-query-editor.rst [hunk 8]
```diff
@@ -19,3 +19,8 @@ Count rows::
 Create a new role::
 
     CREATE ROLE accountant
+
+Alternatives
+-------------
+
+Query editor is convenient if you want to run queries directly from the console on behalf of the default user. However, if you want to run requests using different user, or if you expect a large size of the response, you can use :doc:`play <use-play>`, a built-in user interface accessible through HTTPS.
```
Comment (dewan-ahmed) [inline docs/products/clickhouse/howto/use-query-editor.rst:26]: The query editor is...

Note: Added "the"

However, if you want to run requests using a different user...

Note: Added "a".

**Review by dewan-ahmed — CHANGES_REQUESTED** [review]
Great addition to the ClickHouse docs. Looking forward to the next revision.

File: docs/products/clickhouse/howto/use-play.rst (hunk 1 above)
Comment (anelook) [reply docs/products/clickhouse/howto/use-play.rst:1]: The play UI is not something we built, but rather what is available with ClickHouse. So I intentionally didn't put "Aiven for ClickHouse® play UI" not to mislead that this is part of our product.

Comment (anelook) [reply docs/products/clickhouse/howto/use-play.rst:1]: Same for another entry of "ClickHouse"

File: docs/products/clickhouse/howto/use-play.rst (hunk 3 above)
Comment (anelook) [reply docs/products/clickhouse/howto/use-play.rst]: I still added it. I think it makes sense, when we talk about our service and not ClickHouse open source project

**Comment by anelook** [comment]
@dewan-ahmed, thanks a lot! Used your suggestions everywhere except of the first comment. Let me know if you want to talk about it in detail.

**Comment by anelook** [comment]
I decided to remove `ClickHouse` from the header completely, we usually don't keep the name of the product there at all, and this removes the question if it should be `Aiven for ClickHouse` or `ClickHouse`.

**Review by dewan-ahmed — APPROVED** [review]
This is great. Thanks so much @anelook for this addition to the Aiven for ClickHouse docs.

**Pull request merged**